battery_parameters_test.cpp

```diff
@@ -0,0 +1,20 @@
+#define CATCH_CONFIG_MAIN
+#include "test/catch.hpp"
+#include"battery_parameters.h"
+
+TEST_CASE(" Parameter values"){
+ 
+	int minTempVal = 0;   
+	int maxTempVal = 45;
+	int minSOCVal = 20;   
+	int maxSOCVal = 80;
+  std::string parameterName1 = "Temperature";
+  std::string parameterName2 = "SOC";
+  std::string wrongParameterName = "Charging Rate";
+  
+	BMSParameters obj_BMSParameters;
+  
+  REQUIRE(obj_BMSParameters.sendParameterValuesToConsole(parameterName1, minTempVal, maxTempVal) == true);
```

Comment:
No tests for the output format? This is the end responsibility of the Sender.
The absence of tests here means that future mistakes in the output will only be caught when integrating with the receiver. That will take a lot more time than a simple unit-test